assets/onboarding/log-event.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Send log event.
+ *
+ * @param {string} eventName Event name.
+ * @param {Array} properties Event properties.
+ */
+export const logEvent = ( eventName, properties ) => {
+	window.sensei_log_event( eventName, properties );
+};
+
+/**
+ * Enable or disable event logging.
+ *
+ * @param {boolean} enabled Enabled state.
+ */
+logEvent.enable = ( enabled ) => {
+	window.sensei_event_logging.enabled = enabled;
+};
+
+/**
+ * Send log event when link is opened.
+ *
+ * @param {string} eventName Event name.
+ * @param {Array} properties Event properties.
+ * @return {Object} Element attributes.
+ */
+export const logLink = ( eventName, properties ) => {
```

Comment:
In the `event-logging.js` is a function that handles log in `<a>` with the attribute `data-sensei-log-event`. I think it has the same purpose, right? If yes, would it make sense to use the attribute?

Reply:
I considered it, but relying on the jQuery event handler picking up some attributes rendered by React seemed a bit risky, and didn't want to mix the two worlds. This one also handles middle clicking the link.